index.html

```diff
@@ -35,12 +35,12 @@
         </item>
         <item>
           <h3>LÖVE-API</h3>
-          <a href="https://github.com/love2d-community/love-api"><img src="images/love-api.png"></img></a>
+          <a href="http://love2d-community.github.io/love-api"><img src="images/love-api.png"></img></a>
```

Comment:
Used a http (not secured) because that's link written on github.com/love2d-community/love-api page. 
I feel insecure, but i know too little about net protocols to make another decision